client/landing/stepper/index.tsx

```diff
@@ -114,14 +104,12 @@ window.AppBoot = async () => {
 	// Add accessible-focus listener.
 	accessibleFocus();
 
-	await loadPersistedState();
 	const user = ( await initializeCurrentUser() ) as unknown;
 	const userId = ( user as CurrentUser ).ID;
 
-	const queryClient = createQueryClient();
-	await hydrateBrowserState( queryClient, userId );
+	const queryClient = await createQueryClient( userId );
 
-	initializeAnalytics( user, generateGetSuperProps() );
+	initializeAnalytics( user, getGenericSuperPropsGetter( config ) );
```

Comment:
Note to self: `generateGetSuperProps()` was moved to `getGenericSuperPropsGetter()` under `@automattic/calypso-analytics` in this same PR.